TranslatorReasonerAPI.yaml

```diff
@@ -1022,11 +1024,48 @@ components:
           items:
             $ref: '#/components/schemas/MetaAttribute'
           nullable: true
+        qualifiers:
+          description: >-
+            Qualifiers that are possible to be found on this edge type.
+          items:
+            $ref: '#/components/schemas/MetaQualifier'
+          nullable: true
+          type: array
+        association:
+          type: object
+          $ref: '#/components/schemas/BiolinkEntity'
+          description: >-
+            The Biolink association type (entity) that this edge represents.
+            Associations are classes in Biolink
+            that represent a relationship between two entities.
+            For example, the association 'gene interacts with gene'
+            is represented by the Biolink class,
+            'biolink:GeneToGeneAssociation'.  If association
+            is filled out, then the testing harness can
+            help validate that the qualifiers are being used
+            correctly.
+          example: biolink:ChemicalToGeneAssociation
       required:
         - subject
         - predicate
         - object
       additionalProperties: false
+    MetaQualifier:
+      type: object
+      properties:
+        qualifier_type_id:
```

Comment:
Should qualifier_type_id be required and not nullable here? Would it make sense to make a missing or null qualifier_type_id?

Reply:
fixed